definitions/infra-awselasticacheredisshard/golden_metrics.yml

```diff
@@ -0,0 +1,10 @@
+engineCpuUtilization:
+  title: Engine CPU utilization (%)
+  unit: PERCENTAGE
+  queries:
+    aws:
```

Comment:
are the metrics only reported as dimensional metrics or could they be reported as events too?

Reply:
These ones are reported only as dimensional metrics